promscale/installation/recomm-guide.md

```diff
@@ -0,0 +1,40 @@
+# Resource recommendation guide
+
+The resource recommendation guide is to help you understand the compute and disk requirements for Promscale based on your ingest rate and retention. 
+
+## Metrics
+
+### Prometheus remote write
+
+Below are **compute recommendations** for the Promscale connector and TimescaleDB:
```

Comment:
```suggestion
Compute recommendations for the Promscale connector and TimescaleDB are:
```
https://developers.google.com/style/word-list?hl=en does not recommend using below for the position in the document.